dali/python/nvidia/dali/math.py

```diff
@@ -21,32 +21,161 @@ def _arithm_op(*args, **kwargs):
+    return _arithm_op("cos", input)
 
-def log(value):
-    """Fills the output with logarithm (base-10) of value.
-    :rtype: TensorList of log(value). If value is an integer, the result will be float, otherwise the type is preserved.
+def tan(input):
+    """Fills the output with tangent of the input.
+    :rtype: TensorList of tan(input). If input is an integer, the result will be float, otherwise the type is preserved.
     """
-    return _arithm_op("log", value)
+    return _arithm_op("tan", input)
+
+def asin(input):
+    """Fills the output with arcus sine of the input.
+    :rtype: TensorList of asin(input). If input is an integer, the result will be float, otherwise the type is preserved.
+    """
+    return _arithm_op("asin", input)
+
+def acos(input):
+    """Fills the output with arcus cosine of the input.
+    :rtype: TensorList of acos(input). If input is an integer, the result will be float, otherwise the type is preserved.
+    """
+    return _arithm_op("acos", input)
+
+def atan(input):
+    """Fills the output with arcus tangent of the input.
+    :rtype: TensorList of atan(input). If input is an integer, the result will be float, otherwise the type is preserved.
+    """
+    return _arithm_op("atan", input)
+
+def sinh(input):
+    """Fills the output with hyperbolic sine of the input.
+    :rtype: TensorList of sinh(input). If input is an integer, the result will be float, otherwise the type is preserved.
+    """
+    return _arithm_op("sinh", input)
+
+def cosh(input):
+    """Fills the output with hyperbolic cosine of the input.
+    :rtype: TensorList of cosh(input). If input is an integer, the result will be float, otherwise the type is preserved.
+    """
+    return _arithm_op("cosh", input)
+
+def tanh(input):
+    """Fills the output with hyperbolic tangent of the input.
+    :rtype: TensorList of tanh(input). If input is an integer, the result will be float, otherwise the type is preserved.
+    """
+    return _arithm_op("tanh", input)
+
+def asinh(input):
+    """Fills the output with inverse hyperbolic sine of the input.
+    :rtype: TensorList of asinh(input). If input is an integer, the result will be float, otherwise the type is preserved.
+    """
+    return _arithm_op("asinh", input)
+
+def acosh(input):
+    """Fills the output with inverse hyperbolic cosine of the input.
+    :rtype: TensorList of acosh(input). If input is an integer, the result will be float, otherwise the type is preserved.
+    """
+    return _arithm_op("acosh", input)
+
+def atanh(input):
+    """Fills the output with inverse hyperbolic tangent of the input.
+    :rtype: TensorList of atanh(input). If input is an integer, the result will be float, otherwise the type is preserved.
+    """
+    return _arithm_op("atanh", input)
 
 def min(left, right):
-    """Fills the output with minima of corresponding values in ``left`` and ``right``.
+    """Fills the output with minima of corresponding inputs in ``left`` and ``right``.
```

Comment:
The previous description was _much_ easier to understand (and more precise, and generally better).

Reply:
Fixed